docs/application/web/api/5.5/ui_fw_api/Mobile_UIComponents/mobile_Coverflow.htm

```diff
@@ -0,0 +1,224 @@
+				<td class="option">string | null</td>
+				<td class="option">"coverflow"</td>
+				<td class="option">Effect type.</td>
+			</tr>
+
+		</tbody>
+	</table>
+
+
+
+<h2><a id="methods-coverflow-mobile"></a>Methods</h2>
+
+<p>To call a method on the component, use one of the existing APIs:</p>
+
+<pre class="prettyprint">&lt;div class=&quot;ui-coverflow&quot;&gt;&lt;/div&gt;
+&lt;script&gt;
+    var coverflowElement = document.getElementsByClassName(&quot;ui-coverflow&quot;),
+        coverflow = tau.widget.CoverFlow(coverflowElement);
+
+    coverflow.value(&quot;carousel&quot;);
+&lt;/script&gt;
+</pre>
+
+	<h3>Summary</h3>
+	<table class="informaltable">
+		<thead>
+		<tr>
+			<th>Method</th>
+			<th>Description</th>
+		</tr>
+		</thead>
+		<tbody>
+
+
+				<tr>
+					<td>
+						<pre class="intable prettyprint">string <a href="#method-value">value</a>(string? value) </pre>
+					</td>
+					<td><p>Gets or sets a value.</p></td>
+				</tr>
+
+
+		</tbody>
+	</table>
+	<dl>
+
+
+
+				<dt class="method" id="addidp28072"><code><b><span class="methodName"
+																   id="method-value">value</span></b></code></dt>
+				<dd>
+					<div class="brief">
+						<p>Gets or sets a value.</p>
+					</div>
+					<div class="synopsis">
+						<pre class="signature prettyprint">string value(string? value) </pre>
+					</div>
+
+						<p><span class="version">Since: </span>
+							5.5
+						</p>
+
+					<div class="description">
+						<p>
+							<p>The method returns the inner text of the coverflow effect or sets the text of the coverflow effect.</p>
+
+
+						</p>
+					</div>
+
+					<div class="parameters">
+						<p><span class="param">Parameters:</span></p>
+					<table>
+					<tbody>
+					<tr>
+						<th>Parameter</th>
+						<th>Type</th>
+						<th>Required/optional</th>
+						<th>Default value</th>
+						<th>Description</th>
+					</tr>
+
+
+					<tr>
+						<td><span style="font-family: Courier New,Courier,monospace">value</span></td>
+						<td>string</td>
+						<td>Optional</td>
+						<td>"coverflow"</td>
+						<td>Text to set for the coverflow effect.<br>
+                            Choose "coverflow", "carousel", "wheel", "flat" coverflow effect.</td>
+					</tr>
+
+
+					</tbody>
+					</table>
+					</div>
+
+					<div class="returntype">
+						<p><span class="return">Return value:</span></p>
+
+							<table>
+								<tbody>
+								<tr>
+									<th>Type</th>
+									<th>Description</th>
+								</tr>
+								<tr>
+									<td>string</td>
+									<td>In the get mode, returns the inner text of the coverflow effect.</td>
+								</tr>
+								</tbody>
+							</table>
+
+
+					</div>
+
+						<div class="example">
+							<span class="example"><p>Code
+								example:</p><p></p></span>
+							<pre name="code" class="examplecode
+							prettyprint">
+&lt;div class=&quot;ui-coverflow&quot;&gt;&lt;/div&gt;
+&lt;script&gt;
```

Comment:
Is it Code example:
If so , how about to add in the example code?
<script type="text/javascript" src="lib/tau/libs/tau.coverflow.js"></script>

Reply:
ok. I added.

done.